postgres-dockerfile/Dockerfile

```diff
@@ -20,8 +20,8 @@ RUN apt-get update && \
         pkg-config \
         postgresql-server-dev-${POSTGRES_VERSION}
 
-RUN git clone https://github.com/metabrainz/postgresql-musicbrainz-unaccent.git /tmp/postgresql-musicbrainz-unaccent && \
-    git clone https://github.com/metabrainz/postgresql-musicbrainz-collate.git /tmp/postgresql-musicbrainz-collate
+RUN git clone --depth=1 https://github.com/metabrainz/postgresql-musicbrainz-unaccent.git /tmp/postgresql-musicbrainz-unaccent && \
+    git clone --depth=1 https://github.com/metabrainz/postgresql-musicbrainz-collate.git /tmp/postgresql-musicbrainz-collate
```

Comment:
These changes should apply to https://github.com/metabrainz/musicbrainz-docker/blob/mbvm-38-dev/build/postgres/Dockerfile#L25-L27 instead.